tests/files/locations/test_gcs.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from astro.files.locations import location_factory
+
+
+def describe_is_valid_path():
+    """test is_valid_path() method"""
+
+    def with_supported_filepaths():  # skipcq: PTC-W0065, PYL-W0612
+        """With supported file paths"""
+        filepath = "gs://bucket/some-file"
+        location = location_factory(filepath)
+        assert location.is_valid_path(filepath) is True
```

Comment:
I think we can use parameterize over here instead of describe to test both Positive and negative cases here

Reply:
Yes, this is kinda static check on paths and is implemented in base class, moved it back to it's own test file - test_base.py with parameterized cases.